lisp/ess-custom.el

```diff
@@ -552,7 +552,7 @@ buffer or end chunks etc.")
   :group 'ess-S
   :type 'string)
 
-(defcustom ess-smart-S-assign-key "_"
+(defcustom ess-smart-S-assign-key ?_
```

Comment:
Does the new value actually works?  All changes to the defaults should be checked. I think that in this case it would be more reasonable to do an `(or string character)` as people might have already had customized this.

Any ideas why travis fails?